awx/ui/src/screens/Job/JobDetail/JobDetail.js

```diff
@@ -241,6 +249,11 @@ function JobDetail({ job, inventorySourceLabels }) {
           label={t`Job Type`}
           value={jobTypes[job.type]}
         />
+        <Detail
+          dataCy="source-control-type"
+          label={t`Source Control Type`}
+          value={scmTypes[job.scm_type]}
```

Comment:
does this need to be `scmTypes[job.scm_type] || scmTypes['']`, or will `job.scm_type` always be an empty string if it's not one of the four specific types?

Reply:
I don't think so. Not every `job` will have a `scm_type`, so in those cases we wouldn't want to display the Source Control Type detail.